src/main/scala/com/typesafe/sbt/site/SitePreviewPlugin.scala

```diff
@@ -13,6 +13,7 @@ object SitePreviewPlugin extends AutoPlugin {
     val previewAuto = TaskKey[Unit]("previewAuto", "Launches an automatic jetty server that serves your generated site from the target directory")
     val previewFixedPort = SettingKey[Option[Int]]("previewFixedPort") in previewSite
     val previewLaunchBrowser = SettingKey[Boolean]("previewLaunchBrowser") in previewSite
+    val previewPath = SettingKey[String]("previewPath", "path to open on `previewSite` and `previewAuto`") in previewSite
```

Comment:
Its a bit strange that these settings are defined as `in previewSite` but then are used in both previewSite and previewAuto. But that is for later.

Reply:
https://github.com/sbt/sbt-site/issues/149